lib/options.c

```diff
@@ -316,16 +291,13 @@ get_common_opts (int                    *argc_param,
     while ((c = getopt_long (argc, argv, arg_str, long_options, &option_index))
            != -1)
     {
-        switch (c) {
-        case 1: { /* positional arguments */
-            char **tmp = append_arg_to_vector (&new_argc, new_argv, optarg);
-            if (tmp == NULL) {
-                free(new_argv);
-                return 2;
-            }
-            new_argv = tmp;
+        if (new_argc >= argc) {
```

Comment:
if (new_argc > argc) I believe, since you could execute a command without any common options.

Reply:
Yes equality should should be allowed.